pudb/debugger.py

```diff
@@ -1103,6 +1123,35 @@ def examine_frame(w, size, key):
 
         self.stack_list.listen("enter", examine_frame)
 
+        def open_file_editor(file_name, line_number):
+            try:
+                filename_edited = self.debugger.open_file_to_edit(file_name,
+                                                                  line_number)
+            except Exception:
+                from pudb.lowlevel import format_exception
+                self.message("Exception happened when trying to edit the file:"
+                             "\n\n%s" % ("".join(format_exception(sys.exc_info()))),
+                    title="File Edit Error")
+                return
+
+            self.message("File is changed, but the execution is continued with"
+                         " the 'old' codebase.\n"
+                         f"Changed file: {filename_edited}\n\n"
+                         "Please quit and restart to see changes",
+                         title="File is changed")
```

Comment:
This can claim that the code was changed, when it didn't, which might spook users. Maybe check the mtime with `stat`?

Reply:
And what would be the threshold where we can be certain there was a change? 1minute,  5 minutes?
Even if it is 30 minutes, sometimes I modify code for >30mins, even drink coffee, etc.

I just saying we cannot expect the users to quickly modify code when they are debugging.

What if the message would say something like

> You were in edit mode. If anything was changed in the code, please restart the debugging session, if not, you can continue in this session.

Reply:
I'm not sure I understand. If the mtime is different than before we started the editor, it's pretty clear that the file was changed.

Reply:
Sorry, I overthinked this one. Now it is implemented as requested.

DONE